sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -556,15 +556,49 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
    * For example:
    * {{{
    *   CREATE TABLE [IF NOT EXISTS] [db_name.]table_name
-   *   LIKE [other_db_name.]existing_table_name [USING provider] [locationSpec]
+   *   LIKE [other_db_name.]existing_table_name
+   *   [ROW FORMAT row_format]
+   *   [USING provider | STORED AS hiveFormat]
```

Comment:
nit: `[USING provider | [STORED AS hiveFormat | ROW FORMAT row_format]]` is more precise?

Reply:
How about this?
```
  [USING provider |
    [
      [ROW FORMAT row_format]
      [STORED AS file_format]
         | STORED BY 'storage.handler.class.name' [WITH SERDEPROPERTIES (...)]
    ]
  ]
```